MekHQ/src/mekhq/campaign/personnel/education/EducationController.java

```diff
@@ -454,11 +454,11 @@ private static boolean ongoingEducation(Campaign campaign, Person person, Academ
      * @param person the person to determine graduation for
      * @param academy the academy to determine graduation from
      * @param resources the resources to use for graduation
-     * @return true if the person successfully graduates; otherwise, false
+     * @return true, if the person successfully graduates; otherwise, false
      */
     private static boolean graduationPicker(Campaign campaign, Person person, Academy academy, ResourceBundle resources) {
         if ((academy.isClan()) && (academy.isPrepSchool())) {
-            if (person.getAge(campaign.getLocalDate()) < 10) {
+            if (person.getAge(campaign.getLocalDate()) == 10) {
```

Comment:
Is this right? Wouldn't this make it call graduateClanSibko on those below age 10?
If that _is_ right, I think the logic is unintuitive enough to warrant an extra comment.

Reply:
Persons under the age of 10 shouldn't be graduating. However, as the user can force a graduation that'll cause problems. I'll go ahead and add an extra clause. Thanks for pointing this out!

Reply:
Fixed :)